hearth/sig/lib/hearth/configuration.rbs

```diff
@@ -1,7 +1,7 @@
 module Hearth
-  module Configuration[T]
+  module Configuration
     def initialize: (**Hash[Symbol, untyped] options) -> void
 
-    def merge: (Hash[Symbol, untyped] configuration) -> T
+    def merge: (Hash[Symbol, untyped] configuration) -> Configuration
```

Comment:
I'm pretty sure this still needs to be parameterized - this does not return a Configuration - it returns the service's Config class.